python/tank/bootstrap/resolver.py

```diff
@@ -255,31 +264,48 @@ def resolve_shotgun_configuration(
             # resolve primary and user config
             primary_config = None
             user_config = None
+            secondary_primary_config = None
+            secondary_user_config = None
```

Comment:
the problem is that it isn't the same as the site config.... it's just got the same `None` value...